src/main/webapp/index.html

```diff
@@ -1,14 +1,38 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body >
-      <p>
-          Hello World!!
-      </p>
-  </body>
+    <head>
+        <meta charset="UTF-8">
+        <title>PromoTube</title>
+        <link rel="stylesheet" href="style.css">
+            <script src="script.js"></script>
+    </head>
+    <body >
+        <div class="content">
+        <h1>PromoTube</h1>
+        <form id="searchForm">
+            <input type="text" id="formInput" name="formInput" value="Enter Channel URL">
+            <input type="submit" value="Enter">
+        </form>
+
+        <div class="promoCodeList">
+            <table>
+                <tr>
+                    <th>VideoURL</th>
+                    <th>PROMO CODE</th>
+                    <th>Rating</th>
+                    <th><button>Like</button></th>
+                    <th><button>Dislike</button></th>
+                </tr>
+
+                <tr>
+                    <th><a href="youtube.com">youtube.com</a></th>
+                    <th>XXXXX</th>
```

Comment:
nit: add a todo as to what goes here or will go here.  like <!-- TODO(dantebot): Add header --!>

Reply:
I'm confused as to what you mean. That is just an example promo code listing

Reply:
ah okay.  never mind.  Probably next step is to define a stub servlet methods that returns just a constants like

return new Gson("stuff", "stuff", "stuff");

Then you can try updating the FE to call the backend and it should render "stuff", "stuff", "stuff"